packages/select/src/mwc-select.scss

```diff
@@ -79,11 +79,15 @@ mwc-notched-outline {
 
 :host(:not([disabled])) {
   .mdc-select {
+    .mdc-select__selected-text {
+      color: var(--mdc-select-ink-color, rgba(0,0,0,.87));
+    }
+
     &:not(.mdc-select--outlined) {
       @include mixins.container-fill-color(var(--mdc-select-fill-color, #{vars.$fill-color}));
 
       &:not(.mdc-select--invalid) .mdc-select__selected-text {
-        border-bottom-color: var(--mdc-select-idle-line-color, rgb(0, 0, 0, 0.42));
+        border-bottom-color: var(--mdc-select-idle-line-color, rgba(0, 0, 0, 0.42));
```

Comment:
Can we update the fallback values to correspond to their variable defaults? For example, this should be `vars.$bottom-line-idle-color`.

Additionally, there are quite a few places where we should be using mixins like above. Notably, the line ripple is no longer a border on the selected text element, but a `::before` element on `mdc-line-ripple`. Using the mixin `bottom-line-color()` would avoid this type of breaking change.

I'd also encourage the use of the [`theme.prop()` syntax](https://github.com/material-components/material-components-web/tree/master/packages/mdc-theme#mdc-theme-prop-with-css-custom-properties) for vars. It will add a fallback for browsers that don't support custom properties.

Reply:
Converted the immediate ones around these changes. I believe the others would be better suited for another PR. I'll file an issue RN